package.json

```diff
@@ -18,25 +18,25 @@
     "grunt-move": "^1.0.3",
     "grunt-wp-i18n": "^1.0.3",
     "prettier": "npm:wp-prettier@latest",
-    "webpack": "^5.92.1",
+    "webpack": "^5.94.0",
     "webpack-cli": "^4.10.0"
   },
   "scripts": {
     "clear:transients": "docker-compose run --rm wp-cli wp wc tool run clear_transients --user=1",
     "clear:session": "docker-compose run --rm wp-cli wp wc tool run clear_sessions --user=1",
-    "makepot": "yarn grunt makepot",
-    "cssmin": "yarn grunt cssmin",
-    "jsmin": "yarn webpack",
-    "moveJS": "yarn grunt move",
-    "lint": "yarn prettier assets/js --paren-spacing --tab-width 4 --print-width 120 --no-semi --write",
-    "lint:check": "yarn prettier assets/js --paren-spacing --tab-width 4 --print-width 120 --no-semi --debug-check",
-    "webpack:dev": "yarn webpack --mode=development --watch",
-    "build": "yarn makepot && yarn cssmin && yarn lint:check && yarn lint && yarn jsmin && yarn moveJS"
+    "makepot": "pnpm grunt makepot",
+    "cssmin": "pnpm grunt cssmin",
+    "jsmin": "pnpm webpack",
+    "moveJS": "pnpm grunt move",
+    "lint": "pnpm prettier assets/js --paren-spacing --tab-width 4 --print-width 120 --no-semi --write",
+    "lint:check": "pnpm prettier assets/js --paren-spacing --tab-width 4 --print-width 120 --no-semi --debug-check",
+    "webpack:dev": "pnpm webpack --mode=development --watch",
+    "build": "pnpm makepot && pnpm cssmin && pnpm lint:check && pnpm lint && pnpm jsmin && pnpm moveJS"
   },
   "dependencies": {
     "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
-    "@dintero/checkout-web-sdk": "^0.8.8",
-    "grunt-cli": "^1.4.3",
+    "@dintero/checkout-web-sdk": "^0.8.11",
+    "grunt-cli": "^1.5.0",
     "grunt-zip": "^0.18.2"
   }
```

Comment:
Not strictly required, but possibly it would be a good idea to add a "engines" section to this now that we use pnpm?
For example like WooCommerce does themselves [here](https://github.com/woocommerce/woocommerce/blob/trunk/package.json#L7-L10)
```json
"engines": {
	"node": "^20.11.1",
	"pnpm": "9.1.3"
},
```
But with the versions we require instead.